lib/util/git/index.ts

```diff
@@ -344,14 +345,24 @@ export async function getSubmodules(): Promise<string[]> {
   }
 }
 
-export async function cloneSubmodules(shouldClone: boolean): Promise<void> {
+export async function cloneSubmodules(
+  shouldClone: boolean,
+  cloneSubmodulesFilter: string[] = ['*'],
```

Comment:
```suggestion
  cloneSubmodulesFilter: string[] | undefined,
```

we shouldn't set a default value here, but require this arg to be passed, even when `undefined`